net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/Menu/Menu.xml

```diff
@@ -0,0 +1,9 @@
+<menu>
+  <Services>
+    <FreeRADIUS VisibleName="FreeRADIUS" cssClass="fa fa-address-book-o">
```

Comment:
PS: VisibleName is automatic if it matches the node name, so you don't need it at all here.